nexusdialog/src/main/java/com/github/dkharrat/nexusdialog/FormActivity.java

```diff
@@ -24,14 +24,24 @@ public void onCreate(Bundle savedInstanceState) {
         formController = new FormController(this);
         initForm();
 
+        if (savedInstanceState != null && savedInstanceState.containsKey(MODEL_KW)) {
+            FormModel savedModel = (FormModel) savedInstanceState.getSerializable(MODEL_KW);
+            formController.setModel(savedModel);
```

Comment:
Do we also need to persist other state in `FormController`, like the error display validation method? What do you think is the best way to do that?